libvmi/driver/kvm/kvm_events.c

```diff
@@ -231,10 +227,12 @@ call_event_callback(
     vmi_instance_t vmi,
     vmi_event_t *libvmi_event)
 {
-    event_response_t response;
-    vmi->event_callback = 1;
-    response = libvmi_event->callback(vmi, libvmi_event);
-    vmi->event_callback = 0;
+    event_response_t response = VMI_EVENT_RESPONSE_NONE;
```

Comment:
Maybe @Wenzel can weigh in on this. Is it save to set `VMI_EVENT_RESPONSE_NONE` as a default response for all event types?

Reply:
I think we should mimic the same behavior that already exists in the Xen driver.

Reply:
That's simply not possible since pauses are also events and they don't come through if you have single stepping activated, for example. Also, from the user side this will behave the same as the xen driver. At least that was the goal here.